webviz_subsurface/plugins/_group_tree/_callbacks.py

```diff
@@ -41,13 +44,13 @@ def _update_ensemble_options(
         stat_option_value = (
             stat_option_state if stat_option_state is not None else "mean"
         )
-
-        if not grouptreedata.tree_is_equivalent_in_all_real(ensemble):
+        ensemble = ensemble_dict[ensemble_name]
```

Comment:
Change according to comment above, if it is changed? E.g. `group_tree_data` or `ensemble_group_tree_data`?

Reply:
Yes, I think I will just go for `group_tree_data`, to avoid making the name too long.